source/development/packaging/generate-indexer-package.rst

```diff
@@ -0,0 +1,314 @@
+            package and the Makefile.
+      
+         Current folder loadout at this stage:
+      
+         ::
+      
+            artifacts/
+            |-- dist
+            |   |-- wazuh-indexer-min_|WAZUH_CURRENT|_amd64.deb
+            `-- tmp
+                `-- deb
+                    |-- Makefile
+                    |-- data.tar.gz
+                    |-- debmake_install.sh
+                    |-- etc
+                    |-- usr
+                    |-- var
+                    `-- wazuh-indexer-min_|WAZUH_CURRENT|_amd64.deb
+      
+         ``usr``, ``etc`` and ``var`` folders contain ``wazuh-indexer`` files,
+         extracted from ``wazuh-indexer-min-*.deb``. ``Makefile`` and the
+         ``debmake_install`` are copied over from
+         ``wazuh-indexer/distribution/packages/src/deb``. The
+         ``wazuh-indexer-performance-analyzer.service`` file is also copied
+         from the same folder. It is a dependency of the SPEC file.
+      
+      #. Install the plugins using the ``opensearch-plugin`` CLI tool.
+      
+      #. Set up configuration files.
+      
+            Included in ``min-package``. Default files are overwritten.
+      
+      #. Bundle a DEB file with ``debmake`` and the ``Makefile``.
+      
+            ``debmake`` and other dependencies can be installed using the
+            ``provision.sh`` script. The script is invoked by the GitHub
+            Workflow.
+      
+         Current folder loadout at this stage:
+      
+         ::
+      
+            artifacts/
+            |-- artifact_name.txt
+            |-- dist
+            |   |-- wazuh-indexer-min_|WAZUH_CURRENT|_amd64.deb
+            |   `-- wazuh-indexer_|WAZUH_CURRENT|_amd64.deb
+            `-- tmp
+                `-- deb
+                    |-- Makefile
+                    |-- data.tar.gz
+                    |-- debmake_install.sh
+                    |-- etc
+                    |-- usr
+                    |-- var
+                    |-- wazuh-indexer-min_|WAZUH_CURRENT|_amd64.deb
+                    `-- debian/
+                        | -- control
+                        | -- copyright
+                        | -- rules
+                        | -- preinst
+                        | -- prerm
+                        | -- postinst
+      
+      .. _rpm-1:
+
+   .. group-tab:: TAR
+
+      The assembly process for tarballs consists on:
+      
+      #. Extraction of the minimal package
+      #. Bundling of plugins
+      #. Addition of Wazuh configuration files and tooling
+      #. Compression
+      
+      .. code:: console
+      
+         # bash packaging_scripts/assemble.sh -a x64 -d tar -r 1
+      
+      .. _deb-1:
+
+Build and assemble scripts reference
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The packages' generation process is guided through bash scripts. Below
+is a reference of their inputs, outputs and code:
+
+.. code:: console
```

Comment:
This block doesn't seem to be for the terminal (console). Use proper syntax highlighting or `none` for just a code style formatted block.